submissions/emmanuelEsuong/introductionToHtml/index.html

```diff
@@ -0,0 +1,214 @@
+<<!DOCTYPE html>
```

Comment:
Well done on the job done so far.
You need to remove all styles in this file. Check criteria 1 for this task.

Reply:
Thanks for the reviews. 

Please can you kindly explain more? Removing all styles...  I don't really get this.

Reply:
You are using inline styling in your work while the first acceptance criteria is `Develop a product page for your e-commerce store using HTML Only` emphasizes on ONLY.